packages/mobile/src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -61,7 +62,13 @@ const TrendingStackScreen = createStackScreen<TrendingStackParamList>(Stack => (
 ))
 
 const ExploreStackScreen = createStackScreen<ExploreStackParamList>(Stack => (
-  <Stack.Screen name='explore-stack' component={ExploreScreen} />
+  <>
+    <Stack.Screen name='explore-stack' component={ExploreScreen} />
+    <Stack.Screen
+      name='TrendingUnderground'
```

Comment:
should we keep kebab-case here?

Reply:
chatted with sebastian about it at some point, we are gonna move to PascalCase cause that is what react-native examples default to

Reply:
sounds good 👍